guillotina/tests/dbusers/test_manage_users.py

```diff
@@ -114,6 +114,19 @@ async def test_patch_user_data(dbusers_requester, user_data):
         assert resp["email"] == "foobar2@foo.com"
         assert "guillotina.Manager" in resp["roles"]
 
+        # Change password
+        data = {"password": "asdf"}
+        resp, status_code = await requester("PATCH", "/db/guillotina/@users/foobar", data=json.dumps(data))
```

Comment:
There is already the @reset-password endpoint to set the password of your user, this feature would allow root user to set user password. Is this correct ?

Reply:
IMO it's ok that root can set users' password. Also, that a user can change his own password while is logged (without the need to send an email), but in that case maybe we want to check the user know the current password to set a new one.

Reply:
![Screenshot from 2021-01-17 12-18-47](https://user-images.githubusercontent.com/64216/104838934-46d00300-58be-11eb-9eb0-2aa3071e317b.png)

Reply:
this is a facility to allow root user to change passwords. A user can change it's password if it's logged, (though) this is a side effect, I think it's better to just go with tthis, I don't agree on complicating the endpoint checkin the current user password, at the most, disallow using this feature if the user it's not manager.

Reply:
BTW, the feature also fixes the issue around hashing passwords on PATCH operations.